src/lib/libwebaudio.js

```diff
@@ -100,9 +95,15 @@ var LibraryWebAudio = {
 #endif
 #endif
 
+    // AUDIO_CONTEXT_RENDER_SIZE_DEFAULT and AUDIO_CONTEXT_RENDER_SIZE_HARDWARE
+    // into their AudioContextRenderSizeCategory enum, or take the positive int. 
```

Comment:
Wonder if a verb got cut off from this comment, or maybe this was intentionally brief?